Xamarin.PropertyEditing.Windows/Themes/Resources.xaml

```diff
@@ -522,6 +525,30 @@
 		</Setter>
 	</Style>
 
+	<local:CommonBrushToBrushConverter x:Key="BrushConverter" />
+	<local:CommonColorToColorConverter x:Key="ColorConverter" />
+
+	<DataTemplate x:Key="SimpleResourcePreviewItem">
+		<DataTemplate.Resources>
+			<DataTemplate DataType="{x:Type drawing:CommonColor}">
+				<Rectangle>
```

Comment:
This will not show transparency. You should be using `BrushBoxControl`. You should even be able to handle a preview of any brush, in addition to solid colors.